src/website/app/demos/Pagination/examples/pageSizer.js

```diff
@@ -0,0 +1,84 @@
+/* @flow */
+import { Component } from 'react';
+import Pagination from '../../../../../library/Pagination';
+import Table from '../../../../../library/Table';
+import DemoLayout from '../../shared/DemoLayout';
+import { columns, data } from '../shared/data';
+
+export default {
+  id: 'page-sizer',
+  title: 'Page Sizer',
+  description: `Allow users to select the page size with the \`pageSizer\` prop.
+A custom collection of optional page sizes can be provided with the
+\`pageSizes\` prop. The category descriptor (e.g. "minerals", below) within the 
```

Comment:
I don't think that the `category` prop is going to work for i18n/l10n reasons.  We cannot simply assume that other languages can just append the category value.  We need to give the user full control of the entire string, and provide a sensible default.  I'd suggest adding a function to the messages prop to support this case.

Reply:
The user does have full control of the string with `messages.pageSizer.status`; we thought that additionally pulling out the `category` string (which is utilized in `status`) would be useful for English speakers who only want to change the category displayed (otherwise they need to redefine all messages).